.github/workflows/checks.yml

```diff
@@ -84,6 +84,7 @@ jobs:
         ports:
           # Maps tcp port 6379 on service container to the host
           - 6379:6379
+    if: ${{ github.actor != 'dependabot[bot]' }}
```

Comment:
Quick question - does it matter where this `if` is in the YAML or will GitHub Actions see it and just skip the whole job if the actor is Dependabot?  I ask because there's still some service setup above and just want to make sure it's not spinning stuff up for no reason. 🙂

Reply:
I put it where it is to religiously follow the google examples.  I can try moving it, hopefully if it's wrong the syntax will be broken and there will be some error message for that.